### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by brechtvdv**
Tests work with Basic CSA 0.0.5

### CODE REVIEW

**Comment by brechtvdv** [comment]
Can/should I rebase these commits?

File: lib/BasicCSA.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ var util = require('util')
  */
 var ResultStream = function (query) { //TODO: , footpathsTo) {
   Transform.call(this, {objectMode: true});
+  this._idCounter = 0; // Used as identifier if connections don't have one
```
Comment (pietercolpaert) [inline lib/BasicCSA.js:15]: What's this? It isn't used further on

Comment (brechtvdv) [inline lib/BasicCSA.js:15]: Oh some old code. I'll remove it

Comment (pietercolpaert) [inline lib/BasicCSA.js:15]: Please, also rebase then. Better not to see this in the history :)

**Comment by pietercolpaert** [comment]
If it's not too much of a hassle, I would love that, yes!

**Comment by pietercolpaert** [comment]
Also, if a file in the tests is not used (e.g., the uncompressed version of a dataset), please do not add it to the repo

**Pull request closed without merging**